.clangd

```diff
@@ -1,2 +1,35 @@
 CompileFlags:
-  CompilationDatabase: build/       # Search build/ directory for compile_commands.json
+  # Search build/ directory for compile_commands.json
+  CompilationDatabase: build
+Diagnostics:
+  # Additional checks that are useful but too noisy to be included in .clang-tidy
+  ClangTidy:
+    Add:
+      - google-*
+      - misc-*
+      - performance-*
+      - readability-*
+    Remove:
+      - google-readability-function-size
+      - misc-no-recursion
+      - misc-unused-parameters
+      - readability-function-*
+      - readability-else-after-return
+      - readability-identifier-length
+      - readability-magic-numbers
+      - readability-redundant-function-ptr-dereference
+      - readability-suspicious-call-argument
```

Comment:
I've already told you that `readability-isolate-declaration` should be removed. Others have told you in [your other PR](https://github.com/neovim/neovim/pull/20446) that they don't like this change. Why haven't you removed it?

Reply:
oops, sorry! I thought I add it in https://github.com/neovim/neovim/commit/b039593ada338d75aa3f0f80ac66aef6b6656995